src/app/private/[uid]/admin/pending-chapters/page.tsx

```diff
@@ -1,5 +1,30 @@
-const PendingChapters = () => {
-  return null;
+import ChapterRequest from "@components/ChapterRequest";
+import { prisma } from "@server/db/client";
+
+const PendingChapters = async () => {
+  // Use prisma to get all pending chapter requests
+  const pendingChapters = await prisma.chapterRequest.findMany();
+  console.log(pendingChapters[0]);
+  // Map every chapter request to a pending chapter component and return!
+  return (
+    <div className="mb-5 grid flex-wrap gap-6 md:grid-cols-2">
+      {pendingChapters.map((pendingChapter) => (
+        <ChapterRequest
```

Comment:
Is there a reason why we need to pass the attributes separately for each `ChapterRequest` instead of using the variable `pendingChapter` directly?